dora/core/client/fs/src/main/java/alluxio/client/file/PrefetchCachePolicy.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file;
+
+import alluxio.conf.Configuration;
+import alluxio.conf.PropertyKey;
+
+/**
+ * The prefetch cache policy to determine the prefetch size.
+ */
+public interface PrefetchCachePolicy {
+  /**
+   * Adds the trace of a read request.
+   * @param pos the position
+   * @param size the size
+   */
+  void addTrace(long pos, int size);
+
+  /**
+   * Called when a read hits the cache.
+   */
+  void onCacheHitRead();
+
+  /**
+   * Called when a read does not hit the cache.
+   */
+  void onCacheMissRead();
+
+  /**
+   * @return the expected prefetch size
+   */
+  long getPrefetchSize();
```

Comment:
I don't think it'll ever be necessary to do a prefetch that's larger than `Integer.MAX_VALUE`. `int` works better with array/buffer indices and read lengths without the need to truncate the integers.

```suggestion
  int getPrefetchSize();
```